dev-libs/liborcus/liborcus-9999.ebuild

```diff
@@ -4,51 +4,56 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7,8} )
-inherit python-single-r1
+inherit python-single-r1 autotools
 
 DESCRIPTION="Standalone file import filter library for spreadsheet documents"
 HOMEPAGE="https://gitlab.com/orcus/orcus/blob/master/README.md"
 
 if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="https://gitlab.com/orcus/orcus.git"
-	inherit git-r3 autotools
+	inherit git-r3
 else
 	SRC_URI="https://kohei.us/files/orcus/src/${P}.tar.xz"
-	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+	# depends on spdlog keywording: https://bugs.gentoo.org/708910
+	#KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+	KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
 LICENSE="MIT"
 SLOT="0/0.15" # based on SONAME of liborcus.so
-IUSE="python +spreadsheet-model static-libs tools"
```

Comment:
it is non-optional now? please describe in git commit.

Reply:
Long story short: No, it's still optional and I just introduced a bug by a typo.
I fixed the ebuild locally.